src/app/modules/ix-forms/components/ix-combobox/ix-combobox.component.scss

```diff
@@ -14,7 +14,7 @@
 
 .input-container {
   background: var(--bg1);
-  border: 1.5px solid var(--lines);
+  border: solid 1.5px var(--lines);
```

Comment:
Previous syntax was correct.

Reply:
Yeah, just for searching purposes all other cases follows this order

Reply:
They are incorrect then :)